src/test/kotlin/apply/application/EvaluationTargetCsvServiceTest.kt

```diff
@@ -0,0 +1,83 @@
+package apply.application
+
+import apply.createEvaluationItem
+import apply.domain.evaluationItem.EvaluationItemRepository
+import apply.utils.CsvGenerator
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.AssertionsForInterfaceTypes.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import support.test.UnitTest
+import java.io.InputStream
+
+@UnitTest
+class EvaluationTargetCsvServiceTest {
+    @MockK
+    private lateinit var evaluationTargetService: EvaluationTargetService
+
+    @MockK
+    private lateinit var evaluationItemRepository: EvaluationItemRepository
+
+    @MockK
+    private lateinit var csvGenerator: CsvGenerator
+
+    private lateinit var evaluationTargetCsvService: EvaluationTargetCsvService
+
+    @BeforeEach
+    internal fun setUp() {
+        evaluationTargetCsvService =
+            EvaluationTargetCsvService(evaluationTargetService, evaluationItemRepository, csvGenerator)
+    }
+
+    @Test
+    fun `평가지 양식에 맞는 평가지 업로드 시 csv 읽기에 성공한다`() {
+        // TODO: 해당 테스트는 파일 읽기를 정상적으로 실행하는지 확인하는용으로, 업로드 기능 구현시 제거되어도 되는 테스트
+        val inputStream = getInputStream("evaluation.csv")
+        val evaluationItems = listOf(
+            createEvaluationItem(maximumScore = 1, position = 1),
+            createEvaluationItem(maximumScore = 2, position = 2),
+            createEvaluationItem(maximumScore = 3, position = 3)
+        )
+
+        every { evaluationItemRepository.findByEvaluationIdOrderByPosition(any()) } returns evaluationItems
+
+        assertDoesNotThrow { evaluationTargetCsvService.updateTarget(inputStream, 1L) }
+    }
```

Comment:
`@ParameterizedTest`를 사용하는 방법도 있습니다.
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-argument-conversion

```suggestion
    @ValueSource(strings = ["src/test/resources/evaluation.csv"])
    @ParameterizedTest
    fun `평가지 양식에 맞는 평가지 업로드 시 csv 읽기에 성공한다`(file: File) {
        // TODO: 해당 테스트는 파일 읽기를 정상적으로 실행하는지 확인하는용으로, 업로드 기능 구현시 제거되어도 되는 테스트
        val evaluationItems = listOf(
            createEvaluationItem(maximumScore = 1, position = 1),
            createEvaluationItem(maximumScore = 2, position = 2),
            createEvaluationItem(maximumScore = 3, position = 3)
        )
        every { evaluationItemRepository.findByEvaluationIdOrderByPosition(any()) } returns evaluationItems
        assertDoesNotThrow { evaluationTargetCsvService.updateTarget(file.inputStream(), 1L) }
    }
```